src/PerformanceMonitor/README.md

```diff
@@ -0,0 +1,30 @@
+It is a plugin to create performance metrics used to benchmark NEO3. You can follow the instructions on [neo-modules](https://github.com/neo-project/neo-modules/blob/master/README.md) to install the plugin.
+
+>Note: 
+> 1. Add the `System.Diagnostics.PerformanceCounter` as a dependency in neo-cli to use the `check disk` command.
+> 2. Install `RpcClient` module to use the `rpc time` command. 
```

Comment:
Maybe also include a note here informing that it's required to start consensus before using `confirmation time` and `payload time` commands

Reply:
I add the note in the Consensus Algorithm part.